### DISCUSSION TITLE
Jupyter Notebook display issues on GitHub

### DISCUSSION THREAD

**Discussion by tomerburg** [community/community#13504] [Other Feature Feedback, Questions, & Ideas]
I’m not sure where (and what category) this would be best to post so if this is not the best place for it, hopefully someone can direct it to where it would be best addressed…

For the last 2 days I’ve noticed issues with Jupyter Notebooks displays on GitHub, specifically with printed and outputted text not showing as it should. Here is a sample notebook I wrote to test this, with the output of what I have on my end below:

https://github.com/tomerburg/sample_scripts/blob/main/test.ipynb

![742A00FE-19DD-4826-9B99-4F0524899A66](https://user-images.githubusercontent.com/23174572/160130843-04797ae5-653c-4286-bcf6-b38dc47a97b3.jpeg)

I get this error both on my laptop and phone, on wifi or on cellular data, on multiple browsers (Firefox, chrome, safari), on every notebook I’ve seen whether posted by myself or another person, notebooks created this week or years ago, and numerous people I’ve asked also saw the same when viewing these notebooks. This leads me to suspect it may be an issue on GitHub’s end, but I’m not sure where to direct this issue. If anyone can help this would be greatly appreciated!

Edit: I should also note the output as it’s supposed to be displayed is embedded in the notebook if you use the “raw” view, but it doesn’t show when using the GitHub default viewer for Jupyter notebooks. If you download these notebooks and view them locally, all output is shown as expected.

**Comment by OSU-CMR** [comment]
I am experiencing the exact same issue. Even the ipynb files in older GitHub repos are exhibiting this issue.

**Reply by ChristopherHhh05** [reply]
It seems like the output issue with GitHub was solved today!

**Comment by minimaxir** [comment]
This is a very recent regression on GitHub's end. Text was working normally in Jupyter Notebooks a couple weeks ago,

This oddly only happens on text printed to console; it doesn't happen on HTML displays or logging text.
